torch/_torch_docs.py

```diff
@@ -4298,6 +4298,32 @@ def merge_dicts(*dicts):
             [2., 1., 4., 2.]])
 """.format(**common_args))
 
+add_docstr(torch.polar,
+           r"""
+polar(input, out=None) -> Tensor
+
+Returns a new tensor in the polar form of the elements of
+:attr:`input`. Magnitude is stored in real value and angle in imageniry.
+
+Args:
+    {input}
+    {out}
+
+Example::
+
+    >>> a = torch.randn(4, dtype=torch.complex64)
+    >>> a
+    tensor([                       (-1.1573 - 1.0926j),
+                                    (0.0899 + 0.3232j),
+                                    (0.9718 + 0.1947j),
+                                    (0.2349 + 0.1463j)], dtype=torch.complex64)
+    >>> a.polar()
```

Comment:
Does it make sense to represent polar representation of complex as `a + bi`?

Reply:
Probably not, wasn't sure how to store it differently in complex tensor, I guess it may be resolved by `ComplexPolarFloat` dtype.